src/Symfony/Component/Form/ButtonBuilder.php

```diff
@@ -324,6 +324,8 @@ public function setDataLocked(bool $locked): static
     /**
      * Unsupported method.
      *
+     * @return never
```

Comment:
we should also consider fixing the other method that throw in this file

Reply:
To be honest, I don't get why any of the methods throwing an exception in this file exist. This class extends nothing that requires them to be in this class.